src/utils/db/hub.ts

```diff
@@ -135,7 +135,8 @@ export async function createHub(
         timeStr ? DateTime.fromSQL(timeStr, { zone: 'utc' }).toISO() : null
     )
 
-    const postgres = createPostgresPool(serverConfig)
+    const configOrDatabaseUrl = serverConfig.DATABASE_URL ? serverConfig.DATABASE_URL : serverConfig
```

Comment:
This will always result in `DATABASE_URL` being used, because it has a non-empty default value (while `POSTHOG_DB_NAME` is null by default).

Reply:
Oh, right, this is annoying :$ - looks like it's not required on main PostHog: https://github.com/PostHog/posthog/blob/master/posthog/settings.py#L369-L376 - and hence the precedence order there.

If we use `POSTHOG_DB_NAME`, while PostHog/posthog uses DATABASE_URL, that's.. not good.

I wonder if we can get around this by removing the default value for `DATABASE_URL` ? Does that break things / workflows elsewhere? (I'm guessing probably yes)

.. or maybe it's a documentation thing? We explicitly mention to use one or the other? Although, I'd prefer to fix this ourselves so things are consistent.

Reply:
Yeah, def not good to have a discrepancy. I suppose the fix would then be removing the default value of `DATABASE_URL` here in the non-test non-dev case, and adjusting `createPostgresPool` for same control flow as in the main repo.